lix/packages/sdk/src/open/openLix.ts

```diff
@@ -171,6 +173,22 @@ export async function openLix(args: {
 		commit: (args: { description: string }) => {
 			return commit({ ...args, db, currentAuthor });
 		},
+		createDiscussion: (args: { changeIds?: string[]; body: string }) => {
+			if (currentAuthor === undefined) {
+				throw new Error("current author not set");
+			}
+			return createDiscussion({
+				...args,
+				db,
+				currentAuthor,
+			});
+		},
+		comment: (args: { parentComment: { id: string }; body: string }) => {
```

Comment:
I would use id directly here too to avoid passing around objects that might or might not be convenient to create.

```suggestion
		comment: (args: { parentCommentId: string; body: string }) => {
```